ATPDocs/remediation-actions.md

```diff
@@ -18,7 +18,8 @@ Those response actions on users are available directly from the user page, the u
 
 The following actions can be performed directly on the on-premises account:
 
-- **Disable user**: This will temporarily prevent a user from logging in to the network. This can help prevent compromised users from moving laterally and attempting to exfiltrate data or further compromise the network.
+- **Disable user in AD**: This will temporarily prevent a user from logging in to the on-premises network. This can help prevent compromised users from moving laterally and attempting to exfiltrate data or further compromise the network.
```

Comment:
```suggestion
- **Disable user in Active Directory**: This will temporarily prevent a user from logging in to the on-premises network. This can help prevent compromised users from moving laterally and attempting to exfiltrate data or further compromise the network.
```